content/sensu-go/5.17/reference/agent.md

```diff
@@ -844,10 +844,9 @@ sensu-agent start --allow-list /etc/sensu/check-allow-list.yaml
 # /etc/sensu/agent.yml example
 allow-list: /etc/sensu/check-allow-list.yaml{{< /highlight >}}
 
-
 | labels     |      |
 -------------|------
-description  | Custom attributes to include with event data that you can access with [event filters][9] and [tokens][27].<br><br>In contrast to annotations, you can use labels to create meaningful collections that you can select with [API response filtering][25] and [sensuctl response filtering][26]. Overusing labels can affect Sensu's internal performance, so we recommend moving complex, non-identifying metadata to annotations.
+description  | Custom attributes to include with event data that you can use for response and dashboard filtering.<br><br>If you include labels in your event data, you can filter [API responses][25], [sensuctl responses][26], and [dashboard views][54] based on them. In other words, labels allow you to create meaningful groupings for your data.<br><br>Limit labels to metadata you need to use for response filtering. For complex, non-identifying metadata that you will *not* need to use in response filtering, use annotations rather than labels.
```

Comment:
```suggestion
description  | Custom attributes to include with event data that you can use for response and dashboard view filtering.<br><br>If you include labels in your event data, you can filter [API responses][25], [sensuctl responses][26], and [dashboard views][54] based on them. In other words, labels allow you to create meaningful groupings for your data.<br><br>Limit labels to metadata you need to use for response filtering. For complex, non-identifying metadata that you will *not* need to use in response filtering, use annotations rather than labels.
```